cli/stop_file_writer.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import sys
+import time
+from datetime import datetime, timedelta
+from time import time as current_time
+
+from cli.start_file_writer import is_empty
+from file_writer_control import JobHandler, JobState, WorkerCommandChannel
+
+
+def cli_parser() -> argparse.Namespace:
+    """
+    Parser for the command line interface.
+    """
+
+    fw_parser = argparse.ArgumentParser(
+        fromfile_prefix_chars="@", description="FileWriter Stopper"
+    )
+
+    fw_parser.add_argument(
+        "-s",
+        "--stop",
+        metavar="stop",
+        type=str,
+        help="Stop FileWriter immediately.",
+    )
+    fw_parser.add_argument(
+        "-sa",
+        "--stop_after",
+        metavar="stop_after",
+        nargs=2,
+        type=str,
+        help="Stop FileWriter after a given time in seconds.",
+    )
+    fw_parser.add_argument(
+        "-b",
+        "--broker",
+        metavar="kafka_broker",
+        type=str,
+        default="localhost:9092",
+        help="Kafka broker port.",
+    )
+    fw_parser.add_argument(
+        "-t",
+        "--topic",
+        metavar="consume_topic",
+        type=str,
+        required=True,
+        help="Name of the Kafka topic to be consumed.",
+    )
+    fw_parser.add_argument(
+        "--timeout",
+        metavar="ack_timeout",
+        type=float,
+        default=5,
+        help="How long to wait for timeout on acknowledgement.",
+    )
+
+    args = fw_parser.parse_args()
+
+    return args
+
+
+def create_job_handler(args: argparse.Namespace, job_id: str) -> JobHandler:
+    host = args.broker
+    topic = args.topic
+    command_channel = WorkerCommandChannel(f"{host}/{topic}")
```

Comment:
As we are moving away from creating jobs in the non-pool mode, I'd suggest using WorkerJobPool  here instead.
This also needs to be done for the CLI start file writer. Could you create a separate ticket for that case?

Reply:
Should we wait until you remove it from the control library altogether? Or this PR should handle that?

Reply:
I think doing it now makes sense. It will be easier for me to remove the non-pool jobs then. :)

Reply:
Using job pool requires another Kafka topic which Jonas referred to as the Kafka topic that the available file-writers are listening to for write jobs. I am not sure if the command topic (that we currently use) can be use a job topic as well. 

Have you used this class before?

Reply:
I think you just need to provide a topic name in the file writer config ini file (which is done by the end-user anyway) and create the topic before running the file writer.

Reply:
So we do not need a special configuration other than defining it? Ok, I had the feeling may be we need something special.

Reply:
According to `job_pool.py in examples`, one just have to provide an extra job-pool topic. Apart from that everything stays the same when we had direct communication via `WorkerCommandChannel`.

Reply:
Great. I was just sceptical if that specific topic should be something special or not (configured in a specific way). But not an issue it seems, will do that.

Reply:
yes it works like a charm.

Reply:
I will merge this and make the pool changes in both CLIs simultaneously in a single PR. that is more proper and easy to test.